### PR TITLE
[MM-57369] Remote users no longer labeled as 'remote' after updates to the System Console Users UI

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary
Add shared user icon for users with remote in systems users

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-57369

#### Screenshots
CleanShot 2024-03-28 at 11 51 53@2x

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/admin_console/system_users/system_users.tsx [hunk 1]
```diff
@@ -272,6 +274,16 @@ function SystemUsers(props: Props) {
                                 title={getDisplayName(info.row.original)}
                             >
                                 {getDisplayName(info.row.original) || ''}
+                                {' '}
```
Comment (devinbinnie) [inline webapp/channels/src/components/admin_console/system_users/system_users.tsx:277]: Do we always want this arbitrary whitespace here? Wouldn't it be better to space the tag properly away from the display name span?

**Review by marianunez — COMMENTED** [review]
@M-ZubairAhmed it seems that we wanted to go with the icon and tooltip instead of the tag? See https://mattermost.atlassian.net/browse/MM-57369?focusedCommentId=163088 cc/ @abhijit-singh

File: webapp/channels/src/components/admin_console/system_users/system_users.tsx (hunk 1 above)
Comment (M-ZubairAhmed) [reply webapp/channels/src/components/admin_console/system_users/system_users.tsx:277]: Yes removed and handled by css

**Review by abhijit-singh — APPROVED** [review]
Looks good to me, thanks @M-ZubairAhmed

**Review by marianunez — APPROVED** [review]
Thanks @M-ZubairAhmed!

**Review by devinbinnie — APPROVED** [review]

**Pull request merged**